packages/@webex/plugin-meetings/test/utils/testUtils.js

```diff
@@ -195,12 +195,19 @@ const delayedTest = (callback, timeout) =>
     }, timeout);
   });
 
-const addMedia = (user) => {
-  const mediaReadyPromises = {
-    local: new Defer(),
-    remoteAudio: new Defer(),
-    remoteVideo: new Defer(),
-  };
+const addMedia = (user, options = {}) => {
+  const mediaReadyPromises = Array.isArray(options.streams)
```

Comment:
maybe we should rename `streams` to `expectedMediaReadyTypes` or something like that to reflect the purpose of it better

Reply:
Changed.